client/reader/feed-stream/index.jsx

```diff
@@ -40,7 +40,7 @@ class FeedStream extends React.Component {
 		const emptyContent = <EmptyContent />;
 		const title = getSiteName( { feed, site } ) || this.props.translate( 'Loading Feed' );
 
-		if ( feed && feed.is_error ) {
+		if ( ( feed && feed.is_error ) || ( site && site.is_error ) ) {
```

Comment:
sometimes the feed doesn't error but the site does.  we want to catch both.

Reply:
we may not. there are cases where the feed is ok but the site is in error. For instance, if a site leaves wpcom and the wpcom site is deleted, we'll get an error for the site, but the feed is just fine and should keep working.

Reply:
hmm. then how do we "disconnect" the site/feed?